mw-webapp/src/logic/waysTable/waysColumns.tsx

```diff
@@ -208,14 +208,14 @@ export const waysColumns = [
      * Header
      */
     header: () => (
-      <>
+      <div>
```

Comment:
What purpose for using div instead of fragment?

Reply:
you are right, my mistake, unnecessary div removed, thanks!